tools/gnc_visualizer/scripts/communications/dds_support.py

```diff
@@ -245,18 +245,18 @@ def __dictionary_to_ekf_msg(self, dic):
         for i in range(0, len(dic["cov_diag"])):
             msg.cov_diag[i] = dic["cov_diag"][i]
 
-        msg.confidence = dic["confidence"]
```

Comment:
This looks like we accidentally changed the spacing here

Reply:
fixed it, thanks for spotting it!